backend/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".src/Server.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.json && node ./build/Server.js",
+    "build-watch": "tsc -w -p tsconfig.json",
+    "clean": "rimraf coverage build tmp",
+    "lint": "eslint . --ext .ts,.tsx",
+    "test": "jest",
+    "test-coverage": "jest --coverage",
+    "test-watch": "jest --watch",
+    "test-watch-silent": "jest --watch --silent"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/polubis/ProjectManagementApp.git"
+  },
+  "author": "Cezary Wozniak",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/polubis/ProjectManagementApp/issues"
+  },
+  "homepage": "https://github.com/polubis/ProjectManagementApp#readme",
+  "devDependencies": {
+    "@types/body-parser": "^1.19.0",
+    "@types/cors": "^2.8.7",
+    "@types/dotenv": "^8.2.0",
+    "@types/express": "^4.17.8",
+    "@types/jest": "^26.0.13",
+    "@types/mongoose": "^5.7.36",
+    "@types/node": "^14.6.3",
+    "@types/websocket": "^1.0.1",
+    "@typescript-eslint/eslint-plugin": "^4.0.1",
+    "@typescript-eslint/parser": "^4.0.1",
+    "eslint": "^7.8.1",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-jest": "^23.20.0",
+    "jest": "^26.4.2",
+    "node": "^14.8.0",
+    "prettier": "^2.1.1",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^26.3.0",
+    "tslint": "^6.1.3",
+    "tsutils": "^3.17.1",
+    "typescript": "^4.0.2"
+  },
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "module-alias": "^2.2.2",
+    "mongodb": "^3.6.1",
+    "mongoose": "^5.10.3",
+    "reflect-metadata": "^0.1.13",
+    "websocket": "^1.0.32"
+  },
+  "_moduleAliases": {
```

Comment:
Is this needed ?

Reply:
Since we don't have webpack, it seems that it's necessary. Check [this](https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353) for any further info.